pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>learnositysdk</groupId>
   <artifactId>learnositysdk</artifactId>
-  <version>0.16.2</version>
+  <version>0.16.3</version>
```

Comment:
Please remove the version bump from this PR. This is done as part of the release process.

Reply:
Hi @flakeparadigm I removed that version bump.